### PR TITLE
[Doc][Android][Minor][PanResponder]add doc for native event in doc

### PR DESCRIPTION
**Pull request by qbig**

### CODE REVIEW

File: Libraries/vendor/react/browser/eventPlugins/PanResponder.js [hunk 1]
```diff
@@ -25,7 +25,24 @@ var currentCentroidY = TouchHistoryMath.currentCentroidY;
  * It provides a predictable wrapper of the responder handlers provided by the
  * [gesture responder system](/react-native/docs/gesture-responder-system.html).
  * For each handler, it provides a new `gestureState` object alongside the
- * normal event.
+ * native event ‘evt’ object:
```
Comment (satya164) [inline Libraries/vendor/react/browser/eventPlugins/PanResponder.js]: Remove the text `evt`. It's just a variable name.

`native event ‘evt’ object` -> `native event object`

File: Libraries/vendor/react/browser/eventPlugins/PanResponder.js [hunk 2]
```diff
@@ -25,7 +25,24 @@ var currentCentroidY = TouchHistoryMath.currentCentroidY;
...
+ *
+ * ```
+ * onPanResponderMove: (evt, gestureState) => {}
```
Comment (satya164) [inline Libraries/vendor/react/browser/eventPlugins/PanResponder.js]: `evt` -> `event`

File: Libraries/vendor/react/browser/eventPlugins/PanResponder.js [hunk 3]
```diff
@@ -25,7 +25,24 @@ var currentCentroidY = TouchHistoryMath.currentCentroidY;
...
+ * ```
+ *
+ * A native event `evt` is a synthetic touch event with the following form:
```
Comment (satya164) [inline Libraries/vendor/react/browser/eventPlugins/PanResponder.js]: `A native event `evt` is a` -> `A native event is a`

**Comment by qbig** [comment]
@satya164  updated :)

File: Libraries/vendor/react/browser/eventPlugins/PanResponder.js [hunk 4]
```diff
@@ -25,7 +25,24 @@ var currentCentroidY = TouchHistoryMath.currentCentroidY;
  * It provides a predictable wrapper of the responder handlers provided by the
  * [gesture responder system](/react-native/docs/gesture-responder-system.html).
  * For each handler, it provides a new `gestureState` object alongside the
- * normal event.
+ * native event object:
+ *
+ * ```
+ * onPanResponderMove: (evt, gestureState) => {}
```
Comment (satya164) [inline Libraries/vendor/react/browser/eventPlugins/PanResponder.js]: Can we change evt  to event here also :)

**Comment by satya164** [comment]
Looking good. Let's merge after the small change :)

File: Libraries/vendor/react/browser/eventPlugins/PanResponder.js (hunk 4 above)
Comment (qbig) [inline Libraries/vendor/react/browser/eventPlugins/PanResponder.js]: you mean "(evt, gestureState) => {}" --> "(event, gestureState) => {}" ?

**Comment by qbig** [comment]
@satya164 updated:)

**Comment by satya164** [comment]
Thanks @qbig :)

**Pull request merged**